Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:lts-buster
+
+LABEL maintainer "Daniel Park <dpark@broadinstitute.org>"
+
+RUN npm install libxmljs
```

Comment:
Sorry, forgot to commit this line! I added it to the package.json, so this shouldnt be necessary now. I'll go ahead and merge this PR and remove this line, ok?